minimint/src/consensus/mod.rs

```diff
@@ -293,6 +293,66 @@ where
         }
     }
 
+    pub fn epoch_history(&self, epoch: u64) -> Option<EpochHistory> {
+        self.db.get_value(&EpochHistoryKey(epoch)).unwrap()
```

Comment:
Can we make the `unwrap` an `expect` that describes why it's ok to do please? (I assume that would be the typical `DB error` that we can't handle)